initiatives/info.yml

```diff
@@ -0,0 +1,57 @@
+---
+resource: budgets
+query:
+  model: true
+  sql: |
+    select 
+        decidim_initiatives.id,
+        decidim_initiatives.id::text as id_as_text,
+        title->>'fr' as title,
```

Comment:
change `fr` to `$LANGUAGE_CODE`, as we're going to update the `decidim_metabase` repo to adapt to language of the requested database

Reply:
fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)